src/rqt_reconfigure/param_editors.py

```diff
@@ -32,37 +32,22 @@
 #
 # Author: Isaac Saito, Ze'ev Klapow
 
+
+from decimal import Decimal
 import math
 import os
 
 from python_qt_binding import loadUi
 from python_qt_binding.QtCore import Signal, QLocale
 from python_qt_binding.QtGui import QDoubleValidator, QIntValidator
 from python_qt_binding.QtWidgets import QLabel, QMenu, QWidget
-from decimal import Decimal
-import rospkg
-import rospy
+from ament_index_python import get_resource
 
-EDITOR_TYPES = {
-    'bool': 'BooleanEditor',
-    'str': 'StringEditor',
-    'int': 'IntegerEditor',
-    'double': 'DoubleEditor',
-}
+from rclpy.parameter import Parameter
 
 # These .ui files are frequently loaded multiple times. Since file access
 # costs a lot, only load each file once.
-rp = rospkg.RosPack()
-ui_bool = os.path.join(rp.get_path('rqt_reconfigure'), 'resource',
-                       'editor_bool.ui')
-ui_str = os.path.join(rp.get_path('rqt_reconfigure'), 'resource',
-                      'editor_string.ui')
-ui_num = os.path.join(rp.get_path('rqt_reconfigure'), 'resource',
-                      'editor_number.ui')
-ui_int = ui_num
-ui_enum = os.path.join(rp.get_path('rqt_reconfigure'), 'resource',
-                       'editor_enum.ui')
-
+_, package_path = get_resource('packages', 'rqt_graph')
```

Comment:
This is causing errors every time I try to connect to a node.
```suggestion
_, package_path = get_resource('packages', 'rqt_reconfigure')
```